config/manager/kustomization.yaml

```diff
@@ -1,2 +1,8 @@
 resources:
 - manager.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
this was generated when I called `make deploy`, tbh so far not sure I understand what to do with this...

Reply:
should  we commit it? it will be generated every time

Reply:
we should definitely address this and not commit...
Let's so far just overwrite it each time instead of wiping before each commit and then some time later make it work better. 
Kustomize is still a bit of learning debt to me :(